src/Combinatorics/Matroids/properties.jl

```diff
@@ -912,7 +912,6 @@ function characteristic_polynomial(M::Matroid)
     R, q = polynomial_ring(ZZ, 'q')
     return (-1)^M.pm_matroid.RANK*tutte_polynomial(M)(1-q,0)
 end
-charpoly(M::Matroid) = characteristic_polynomial(M)
```

Comment:
Just above the deleted line, the characteristic_polynomial is defined by means of a method for this particular context (and below it the reduced_characteristic_polynomial for the same context). Have you tested that characteristic_polynomial for matroids is still correctly accessible/exported after the change? (This instance of characteristic_polynomial seems to be in the flavor of a chromatic polynomial, not of a characteristic polynomial in other contexts)